### ISSUE TITLE
create a new ufw lxd application for standalone or clustered lxd servers

### ISSUE THREAD

**Issue by eedgar** [canonical/lxd#12527]
This is an enhancement request to ship a prebuilt ufw application for ubuntu servers.  This maybe is generated based on the port configurations in use and/or networks defined?  perhaps auto detecting subnets that clusters are communicating over.  It would be nice to have firewalls in place that are super easy and enhance security of these clusters.

**Comment by simondeziel** [comment]
@eedgar I think that in terms of integration with `ufw`, the best we can do is provide guidance (like we do in https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#ufw-add-rules-for-the-bridge). The reason being that this really a site specific configuration and it also evolves dynamically (say you reconfigure your listeners or add new ones).

**Comment by eedgar** [comment]
is it really the best we could do though?  I mean we do have information on listeners and networks and information like that .. perhaps the config could be dynamically regenerated .. in any case I have my case working .. it was just a nice to have feature idea.

**Comment by simondeziel** [comment]
> is it really the best we could do though?

Yeah, poor choice of wording on my part, we can almost always do better :)

I guess what I wanted to convey is that it didn't seem trivial to do and I'm not sure how big is the intersection between LXD and `ufw` users.

**Comment by escabo** [comment]
After review, we have decided to stick to documentation at this point.
